airbrus.mt

```diff
@@ -188,7 +244,7 @@ def main(argv, => Timer,
                 def brusHelp := makeAirbrusHelp(fn s {client.say(channel, s)})
                 def instanceEnv := ["&&help" => &&brusHelp]
                 def userEnv := userEnvironments.fetch(user.getNick(),
-                                                      fn {baseEnv | instanceEnv})
+                                                      fn { return baseEnv | instanceEnv | ["giftExchange" => make_giftExchange_interface(user.getNick())] })
```

Comment:
been bitten too many times in various other programming languages.

It is also a holdover from when I was playing with E. (Which does not have implict return iirc)